repo_link/repo_link.py

```diff
@@ -6,6 +6,7 @@
 from pathlib import Path
 from typing import Optional, Union, NamedTuple, Sequence, TypeVar, Callable
 from git import Repo  # type: ignore
+import gitdb
```

Comment:
Is this used?

Reply:
Yes, only the exception should be imported.